src/hotspot/cpu/aarch64/frame_aarch64.inline.hpp

```diff
@@ -314,7 +314,6 @@ inline int frame::interpreter_frame_monitor_size() {
   return BasicObjectLock::size();
 }
 
-
```

Comment:
Nit: please just restore these files that only have a blank line removed. It will shrink the size of the PR for other reviewers. Thanks.

Reply:
Fixed, Thanks for review.